tests/e2e/staging_env_basic.go

```diff
@@ -0,0 +1,1358 @@
+					gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Disk is not attached to the node")
+					gomega.Expect(isDiskAttached).To(gomega.BeTrue(), "Disk is not attached")
+					ginkgo.By("After scale up, verify the attached volumes match those in CNS Cache")
+					err = verifyVolumeMetadataInCNS(&e2eVSphere, pv.Spec.CSI.VolumeHandle,
+						volumespec.PersistentVolumeClaim.ClaimName, pv.ObjectMeta.Name, sspod.Name)
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+					volumesBeforeScaleDown = append(volumesBeforeScaleDown, volumespec.PersistentVolumeClaim.ClaimName)
+				}
+			}
+		}
+		replicas = 0
+		ginkgo.By(fmt.Sprintf("Scaling down statefulsets to number of Replica: %v", replicas))
+		_, scaledownErr = fss.Scale(client, statefulset, replicas)
+		gomega.Expect(scaledownErr).NotTo(gomega.HaveOccurred())
+		fss.WaitForStatusReplicas(client, statefulset, replicas)
+		ssPodsAfterScaleDown = fss.GetPodList(client, statefulset)
+		gomega.Expect(len(ssPodsAfterScaleDown.Items) == int(replicas)).To(gomega.BeTrue(),
+			"Number of Pods in the statefulset should match with number of replicas")
+
+		framework.Logf("Deleting statefulset %v", statefulset.Name)
+		fss.WaitForStatusReadyReplicas(client, statefulset, 0)
+		err = client.AppsV1().StatefulSets(statefulset.Namespace).Delete(context.TODO(),
+			statefulset.Name, metav1.DeleteOptions{OrphanDependents: new(bool)})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Wait and verify PVC is fully deleted")
+		for _, volume := range volumesBeforeScaleDown {
+			ginkgo.By(fmt.Sprintf("Wait and verify PVC is fully deleted %s", volume))
+			err := fpv.DeletePersistentVolumeClaim(client, volume, namespace)
+			if !apierrors.IsNotFound(err) {
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+		}
+	})
+
+	/*
+		verify online volume expansion on statefulset
+			1. Create a SC with allowVolumeExpansion set to 'true' in SVC
+			2. create statefulset with replica 3 using the above created SC
+			3. Once all the statefull set PODs are up follow the below step to edit statefulset
+			4. kubectl edit pvc <pvcName>  for each PVC in the StatefulSet, to increase its capacity.
+			5. kubectl delete sts --cascade=false <statefullSetName>  to delete the StatefulSet and leave its pods.
+			6. vi statefulset.yaml and edit the storage and increase the size to the size you have edited the PVC in step4
+			7. create the same statefulset again
+			8. Scaleup statefulset
+			9. Newly created statefulset should have the increased size
+			10. scale down statefulset to 0
+			11. delete statefulset and all PVC's and SC's
+	*/
+	ginkgo.It("Staging Verify online volume expansion on statefulset", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var pvcSizeBeforeExpansion int64
+
+		// Generate a random name for Statefulset app
+		min := 82
+		max := 90
+		port := int32(rand.Intn(max-min) + min)
```

Comment:
use port numbers above 1023 (known ports), preferably between 32768 - 65535

Reply:
Updated the min and max port numbers